build.gradle

```diff
@@ -160,6 +160,10 @@ subprojects {
         dependsOn project.javadoc
     }
     if (!sourceSets.test.allSource.files.isEmpty()) {
+        dependencies {
+            // Add dependency on testHelperFixtures for all projects with tests
+            testImplementation(testFixtures(project(":testHelperFixtures")))
```

Comment:
Instead could we add this to `buildSrc/src/main/groovy/org.opensearch.migrations.java-common-conventions.gradle`?